### PR TITLE
Kt 2021 proposalitem features conditions

### PR DESCRIPTION
**Pull request by khamui**

### CODE REVIEW

File: meinberlin/apps/budgeting/assets/BudgetingProposalList.jsx [hunk 1]
```diff
@@ -31,21 +31,24 @@ export const BudgetingProposalList = (props) => {
...
-              proposal={proposal}
-            />)}
-        </ul>
-        {meta?.is_paginated &&
-          <Pagination
-            currPageIndex={meta.current_page}
-            pageCount={meta.page_count}
-            onPaginate={newUrl => onPaginate(newUrl)}
-          />}
+    <div class="module-content--light">
```
Comment (phillimorland) [inline meinberlin/apps/budgeting/assets/BudgetingProposalList.jsx:34]: className ;)

**Review by phillimorland — APPROVED** [review]
Very nice, just that typo :)

File: meinberlin/apps/budgeting/assets/BudgetingProposalListItem.jest.jsx [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+    name: 'myProposal',
+    url: 'www',
+    creator: 'creator',
+    created: '2021-11-11T15:37:19.490201+01:00'
+  }
+  render(<BudgetingProposalListItem proposal={proposal} isVotingPhase={false} />)
+  expect(screen.getByText('myProposal')).toBeTruthy()
+  expect(screen.getByText('creator')).toBeTruthy()
+  expect(screen.getByText('11. mock text 2021')).toBeTruthy()
+})
```
Comment (phillimorland) [inline meinberlin/apps/budgeting/assets/BudgetingProposalListItem.jest.jsx:29]: @khamui oh and can we keep putting tests in __tests__ folder like the others, it's much easier to see whats what?

Comment (khamui) [reply meinberlin/apps/budgeting/assets/BudgetingProposalListItem.jest.jsx:29]: ah oh, yes of course. I forgot about that..will fix

**Pull request merged**